Brokerages/Backtesting/OandaBrokerageModel.cs

```diff
@@ -0,0 +1,64 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+using System;
+using QuantConnect.Orders;
+using QuantConnect.Securities;
+using QuantConnect.Securities.Interfaces;
+
+namespace QuantConnect.Brokerages.Backtesting
+{
+    /// <summary>
+    ///     Oanda Brokerage Model Implemenatation for Back Testing.
+    /// </summary>
+    public class OandaBrokerageModel : DefaultBrokerageModel
+    {
+        /// <summary>
+        ///     Returns true if the brokerage could accept this order. This takes into account
+        ///     order type, security type, and order size limits.
+        /// </summary>
+        /// <remarks>
+        ///     For example, a brokerage may have no connectivity at certain times, or an order rate/size limit
+        /// </remarks>
+        /// <param name="security">The security being ordered</param>
+        /// <param name="order">The order to be processed</param>
+        /// <param name="message">If this function returns false, a brokerage message detailing why the order may not be submitted</param>
+        /// <returns>True if the brokerage could process the order, false otherwise</returns>
+        public override bool CanSubmitOrder(Security security, Order order, out BrokerageMessageEvent message)
```

Comment:
I think we may want to override CanExecuteOrder for this behavior. [CanSubmitOrder](https://github.com/QuantConnect/Lean/blob/master/Engine/TransactionHandlers/BrokerageTransactionHandler.cs#L593) is called pre-submit to the brokerage, whereas [CanExecuteOrder](https://github.com/QuantConnect/Lean/blob/master/Brokerages/Backtesting/BacktestingBrokerage.cs#L221) is called each time an order is checked for a fill in backtesting.